tests/unit/classifier/svm/LibLinear_unittest.cc

```diff
@@ -6,1292 +6,469 @@
  *
  * Written (W) 2014 pl8787
  */
+#include <gtest/gtest.h>
+#include "utils/Utils.h"
 
 #include <shogun/classifier/svm/LibLinear.h>
 #include <shogun/features/DataGenerator.h>
 #include <shogun/features/DenseFeatures.h>
 #include <shogun/evaluation/ContingencyTableEvaluation.h>
 #include <shogun/mathematics/Math.h>
 #include <gtest/gtest.h>
+#include <map>
+#include <string>
+#include <vector>
 
 using namespace shogun;
 
 #ifdef HAVE_LAPACK
```

Comment:
i think this can go as this was required for the gaussian blob generator, but that's not LAPACK only anymore...

Reply:
I used the string and vector in the tests, so I will only delete <map>, is that okay?

Reply:
lapack can still go imo, no?